pages/dkp/kommander/2.1/operations/access-control/index.md

```diff
@@ -67,20 +67,23 @@ subjects:
 
 ## Types of Access Control Objects
 
-Kubernetes role-based access control can be controlled with three different object categories: Groups, Roles and Policies, as explained in more detail below.
+Kubernetes role-based access control can be controlled with three different object categories: Groups, Roles, and Role Bindings, as explained in more detail below.
 
 ### Groups
 
-You can map group and user claims made by your configured identity providers to Kommander groups by clicking the _Groups_ tab under Administration / Identity providers on the global level.
+You can map group and user claims made by your configured identity providers to Kommander groups by clicking the _Groups_ tab under Administration / Identity providers on the global workspace level.
 
 ![Identity Provider Groups](/dkp/kommander/2.1/img/access-control-idp-groups.png)
 
 ### Roles
 
-ClusterRoles are named collections of rules defining which verbs can be applied to which resources.
+`ClusterRoles` are named collections of rules defining which verbs can be applied to which resources.
```

Comment:
This seems out of place way down here. Seems like this explanation should be offered earlier in this topic.

Reply:
There is a bit of discussion a bit before here - it's at least in this table that's above, so I think there is some start to it.

This topic is ... essentially a few topics inside the overall Access Control area.